### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by olawale-o**
## In this PR, I did the following:
- Create **vet_clinic** database
- Create **animals** table with following columns
  - id: integer
  - name: string
  - date_of_birth: date
  - escape_attempts: integer
  - neutered: boolean
  - weight_kg: decimal
- Insert the following data:
  - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
  - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

- Write queries for the following:
  - Find all animals whose name ends in "mon".
  - List the name of all animals born between 2016 and 2019.
  - List the name of all animals that are neutered and have less than 3 escape attempts.
  - List date of birth of all animals named either "Agumon" or "Pikachu".
  - List name and escape attempts of animals that weigh more than 10.5kg
  - Find all animals that are neutered.
  - Find all animals not named Gabumon.
  - Find all animals with a weight between 10.4kg and 17.3

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -24,4 +24,5 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (Davidosky007) [inline queries.sql:27]: ## [OPTIONAL] 
* This is a better way to write this query `SELECT id, name, date_of_birth, neutered, escape_attempts, date_of_birth, weight_kg FROM
  (SELECT *, EXTRACT(YEAR FROM date_of_birth) AS year FROM animals) 
  AS INNER_TABLE WHERE year BETWEEN 2016 and 2019;` as it is easier to read `SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'`

**Comment by Davidosky007** [comment]
## Hi @olawale-o 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

doing-good-gif
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by olawale-o** [comment]
Hi @Davidosky007 

Thank you for your time and effort on the review

File: queries.sql (hunk 1 above)
Comment (olawale-o) [reply queries.sql:27]: Thank you for the helpful feedback

**Pull request merged**